dwave/system/composites/parallel_embeddings.py

```diff
@@ -0,0 +1,323 @@
+    can be difficult to obtain. Relying on the defaults of this routine may result
+    in slow embedding, see :minorminer.util.parallel_embedding for methods. Note
+    that parallelization of job submissions can mitigate for network latency,
+    programming time and readout time in the case of QPU samplers, subject to
+    additional complexity in the embedding process.
+
+    If a list of embeddings is not provided, the function
+    :code:minorminer.utils.parallel_embeddings.find_multiple_embeddings is called
+    by default to attempt a maximum number of embeddings. If the target and source
+    graph match processor architecture on a Chimera, Pegasus or Zephyr then
+    tiling of a known embedding in a regular pattern may be a useful embedding
+    strategy and find_sublattice_embeddings can be considered.
+    See minorminer.utils.parallel_embedding documentation for customizable options
+    including specification of the time_out and maximum number of embeddings.
+    See tests/test_parallel_embeddings.py for use cases beyond the examples
+    provided.
+
+    Args:
+       sampler (:class:`~dimod.Sampler`): Structured dimod sampler such as a
+           :class:`~dwave.system.samplers.DWaveSampler`.
+       embeddings (list, optional): A list of embeddings. Each embedding is
+           assumed to be a dictionary with source-graph nodes as keys and iterables
+           on target-graph nodes to as values. The embeddings can include keys not
+           required by the source graph. Note that one_to_iterable is ignored
+           (assumed True).
+       source (nx.Graph, optional): A source graph must be provided if embeddings
+           are not specified. The source graph nodes should be supported by
+           every embedding.
+       embedder (Callable, optional): A function that returns
+           embeddings when it is not provided. The first two arguments are
+           assumed to be the source and target graph.
+       embedder_kwargs (dict, optional): key word arguments for the
+           embedder function. The default
+           is an empty dictionary.
+       one_to_iterable (bool, default=False): This parameter should be fixed to
+           match the value type returned by embedder. If False the
+           values in every dictionary are target nodes (defining a subgraph
+           embedding), these are transformed to tuples for compatibility with embed_bqm
+           and unembed_sampleset. If True, the values are iterables over target
+           nodes and no transformation is required.
+
+    Raises:
+        ValueError: If the child sampler is not a structured sampler.
+           If neither embeddings, nor a source graph, are provided.
+           If the embeddings provided are an empty list, or no embeddings are found.
+           If embeddings and source graph nodes are inconsistent.
+           If embeddings and target graph nodes are inconsistent.
+
+    Examples:
```

Comment:
Examples were edited in the wrong place. See the sample() Examples, which will be moved here next commit.